docs/layouts/partials/contribute_list.html

```diff
@@ -0,0 +1,14 @@
+<div class="dropdown" id="contribute-list">
+    <button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownContributeButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><strong>
+        Contribute</strong>
+    </button>
+
+    <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+        <a class="dropdown-item" href="https://github.com/yugabyte/yugabyte-db/issues/new?title=[docs]+{{ $.Page.Title }}&labels=area/documentation&body={{ .Permalink }}%0A%0A%23%23%20Issue%20Description%0A%0A%23%23%20Suggested%20Resolution">
+            <strong>Report Doc Issue</strong>
+        </a>
+        <a class="dropdown-item" href="https://github.com/yugabyte/yugabyte-db/issues/new?title=[docs]+New+Docs+Proposal&labels=area/documentation,community/request&body={{ .Permalink }}%0A%0A%23%23%20Issue%20Description%0A%0A%23%23%20Suggested%20Resolution">
```

Comment:
https://github.com/yugabyte/yugabyte-db/issues/new?title=[docs]+New+docs+proposal&labels=area/documentation,community/request&body=https%3a%2f%2fdocs.yugabyte.com%2flatest%2fexplore%2ffault-tolerance%2fmacos%2f%0A%0A%23%23%20Issue%20description%0A%0A%23%23%20Suggested%20improvement

```suggestion
        <a class="dropdown-item" href="https://github.com/yugabyte/yugabyte-db/issues/new?title=[docs]+New+docs+proposal&labels=area/documentation,community/request&body={{ .Permalink }}%0A%0A%23%23%20Issue%20description%0A%0A%23%23%20Suggested%20improvement">
```